src/engine.ts

```diff
@@ -59,6 +59,7 @@ const defaultPredicates: PredicateRegistry = {
   AdminCommandContainerMissing,
   And,
   AnyOf,
+  AllOf,
```

Comment:
And that this should be before `And`
Not that we seem to be caring about alphabetizing things anymore, but still